castero/episode.py

```diff
@@ -37,7 +38,9 @@ def __init__(self, feed, ep_id=None, title=None, description=None,
         self._copyright = copyright
         self._enclosure = enclosure
         self._played = played
+        self._progress = progress
         self._downloaded = None
+        self.PROGRESS_INDICATOR = "*"
```

Comment:
nitpick: put this inside the class but before any methods (similar to how it's done in the display class)